postmarketOS/pmbootstrap
# ccache: Fix for distcc cross-compiling / various improvements

* Make ccache work when cross-compiling with distcc (fix #716)
* Allow to configure the ccache size in "pmbootstrap init"
* Moved ccache stats code from pmb/build/other.py to
  pmb/helpers/frontend.py
* Grouped job count, ccache size and timestamp based rebuilds
  together to "build options" and allow to skip them
* Sorted config options that had to be modified anyway
  alphabetically

### How to test
* Run `pmbootstrap init` and check out the new grouped options there
  * Change the ccache size there
* Verify that ccache works, and that the cache size gets set properly:
  * `pmbootstrap stats --arch=armhf`
  * `pmbootstrap build --arch=armhf hello-world`
  * `pmbootstrap stats --arch=armhf`
  * `pmbootstrap build --arch=armhf hello-world`
  * `pmbootstrap stats --arch=armhf`
